website/content/docs/east-west/network-area.mdx

```diff
@@ -7,84 +7,340 @@ description: >-
 
 # Federate multiple datacenters with network areas
 
+This topic covers the advanced form of federating Consul datacenters using the network areas capability of Consul Enterprise. For the basic form of federation available in the open source version of Consul, check the [basic federation documentation](/consul/docs/east-west/wan-federation/vms).
+
 <Note title="Enterprise Only">
 
- The network area functionality demonstrated here requires version 0.8.0+ of HashiCorp Cloud Platform (HCP) or self-managed [Consul Enterprise](https://www.hashicorp.com/products/consul/pricing/). If you've purchased or wish to try out Consul Enterprise, refer to [how to access Consul Enterprise](/consul/docs/enterprise#access-consul-enterprise).
+ The network area functionality demonstrated here requires [Consul Enterprise](https://www.hashicorp.com/products/consul/pricing/). If you've purchased or wish to try out Consul Enterprise, refer to [how to access Consul Enterprise](/consul/docs/enterprise#access-consul-enterprise).
 
 </Note>
 
+## Network areas introduction
+
 One of the key features of Consul is its support for multiple datacenters. The architecture of Consul is designed to promote a low coupling of datacenters so that connectivity issues or failure of any datacenter does not impact the availability of Consul in other datacenters. This means each datacenter runs independently, each having a dedicated group of servers and a private LAN gossip pool.
 
-This tutorial covers the advanced form of federating Consul datacenters using the new network areas capability added in Consul Enterprise version 0.8.0. For the basic form of federation available in the open source version of Consul, check the [basic federation tutorial](/consul/docs/east-west/wan-federation/create) for more details.
+Network areas specify a relationship between a pair of Consul datacenters. Operators create reciprocal areas on each side of the relationship and then join them together, so a given Consul datacenter can participate in many areas, even when some of the peer areas cannot contact each other. This allows for more flexible relationships between Consul datacenters, such as hub/spoke or more general tree structures.
+
+<Tip>
+
+Currently, Consul will only route RPC requests to datacenters it is immediately adjacent to via an area, but future versions of Consul may add routing support.
+
+</Tip>
```

Comment:
i would move this after the explanation of the differences between network areas and wan fed. the differences is golden on why i need network areas in the first place